GeneratorInterface/SherpaInterface/src/SherpackUtilities.cc

```diff
@@ -153,6 +154,11 @@ void zerr(int ret)
 /* compress or decompress from stdin to stdout */
 int Unzip(std::string infile, std::string outfile)
 {
+    std::string tmpdir = getenv("TMPDIR");
```

Comment:
hi @perrozzi - could we follow @bbockelm guidance here and first check the size of tmpdir. it seems the maximum total length is 108 for the path - so the check should be just like if (tmpdir.size() > 50 ) { ...} [probably you know better than I how long the sherpa part of that path is going to be ..]

Reply:
I expect the Sherpa directories to be O(10) chars, cfr https://hypernews.cern.ch/HyperNews/CMS/get/edmFramework/3807/1/1/1.html
I will put >50 just to be on the safe side

Reply:
There is no guarantee that `$TMPDIR` is set, meaning that `getenv` could result in a `nullptr`.